README.md

```diff
@@ -36,3 +36,10 @@ for client libraries to work.
     $ grpc_cli call localhost:8080 CreateCustomer \
         "name: 'google' address: 'amphitheatre pkwy'"
     ```
+
```

Comment:
I can also delete these instructions; they might not be necessary. Also, we might change how we run tests soon.